### DISCUSSION TITLE
cannot execute binary file: Exec format error

### DISCUSSION THREAD

**Discussion by alirezaalavi87** [extrawurst/gitui#2138] [General]
Hi.
I feel like I'm doing something wrong. my laptop is 64 bits with x86 architecture linux.
I download the aarch64 binary, run it and the this error: `cannot execute binary file: Exec format error`

what is the issue?

**Comment by extrawurst** [comment]
Please provide more info on what system you are trying to run on

**Comment by aubade** [comment]
Aarch64 is an ARM architecture, and not amd64/x86_64 compatible

(You want to download the one that's just plain musl with no specified CPU arch)

**Reply by extrawurst** [reply]
see #2148

**Comment by alirezaalavi87** [comment]
Ah I see... thanks.
